### PR TITLE
Generate release notes for 2.6.5

### PR DESCRIPTION
**Pull request by madkinsz**
[Rendered](https://github.com/PrefectHQ/prefect/blob/release-notes-2.6.5/RELEASE-NOTES.md#release-265)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
 # Prefect Release Notes
 
+## Release 2.6.5
+
+### Enhancements
+- Add suppport for flow run retries — https://github.com/PrefectHQ/prefect/pull/7152
```
Comment (madkinsz) [inline RELEASE-NOTES.md:6]: This needs QA and may need to be omitted

Comment (madkinsz) [inline RELEASE-NOTES.md:6]:
```suggestion
- Add support for manual flow run retries — https://github.com/PrefectHQ/prefect/pull/7152
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,44 @@
 # Prefect Release Notes
 
+## Release 2.6.5
+
+### Enhancements
+- Add support for manual flow run retries — https://github.com/PrefectHQ/prefect/pull/7152
```
Comment (madkinsz) [inline RELEASE-NOTES.md:6]: This needs QA and may be omitted from this release

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
+- Add persistence of data when a state is returned from a task or flow — https://github.com/PrefectHQ/prefect/pull/7316
+- Add `ignore_file` to `Deployment.build_from_flow()` — https://github.com/PrefectHQ/prefect/pull/7012
+
+### Fixes
+- Allow `with_options` to reset retries and retry delays — https://github.com/PrefectHQ/prefect/pull/7276
+- Fix typo in Results documentation — https://github.com/PrefectHQ/prefect/pull/7324
+- Fix proxy-awareness in the `OrionClient` — https://github.com/PrefectHQ/prefect/pull/7328
+- Fix block auto-registration when changing databases — https://github.com/PrefectHQ/prefect/pull/7350
+- Include hidden files when uploading directories to `RemoteFileSystem` storage — https://github.com/PrefectHQ/prefect/pull/7336
+- UI: Fix breaking color change — https://github.com/PrefectHQ/prefect/pull/7321
```
Comment (madkinsz) [inline RELEASE-NOTES.md:24]: @stackoverfloweth I'm not sure what this means. Can you update this with the user-facing change?

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,44 @@
...
+- Add support for multiple work queue prefixes — https://github.com/PrefectHQ/prefect/pull/7222
+- Include "-" before random suffix of Kubernetes job names — https://github.com/PrefectHQ/prefect/pull/7329
+- Allow a working directory to be specified for `Process` infrastructure — https://github.com/PrefectHQ/prefect/pull/7252
+- Add support for Python 3.11 — https://github.com/PrefectHQ/prefect/pull/7304
+- Add persistence of data when a state is returned from a task or flow — https://github.com/PrefectHQ/prefect/pull/7316
+- Add `ignore_file` to `Deployment.build_from_flow()` — https://github.com/PrefectHQ/prefect/pull/7012
+
+### Fixes
+- Allow `with_options` to reset retries and retry delays — https://github.com/PrefectHQ/prefect/pull/7276
+- Fix typo in Results documentation — https://github.com/PrefectHQ/prefect/pull/7324
```
Comment (madkinsz) [inline RELEASE-NOTES.md:20]:
```suggestion
```

File: RELEASE-NOTES.md (hunk 3 above)
Comment (stackoverfloweth) [reply RELEASE-NOTES.md]: fixed, thanks for running QA here @madkinsz

**Review by anna-geller — APPROVED** [review]

File: RELEASE-NOTES.md (hunk 2 above)
Comment (bunchesofdonald) [reply RELEASE-NOTES.md:6]: QA was done and it looks good.

**Pull request merged**